website/src/components/lifeCycle/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import styles from './styles.module.css';
+
+export default function Lifecycle(props) {
+    return (
+        <span className={styles.lifecycle}>{props.status}</span>
+    );
+}
```

Comment:
We didn't get to this during our meeting today, but I think there's more more improvement we can make with this. As-is, it will always apply the span tag, even if a user forgets to pass in content into the `status` prop. This will cause the pill to still appear with no text within it.

What we can do is check if `props.status` is set before adding the span tag to the page. If not, we should return `null` here rather than returning the `span` tag.

Here's [React's documentation](https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null) on how you can do this.

Reply:
hey @JKarlavige ! thank you and great flag -- i added the null conditional rendering and if a user doesn't add anything, a small '-' appears at the end but i think it's related to the docusaurus' id assignment. 


i also experimented to see if i could pass a props from the frontmatter object (as opposed to the string) and it works too! 

![Screenshot 2023-11-24 at 05 01 47](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/57a90e5c-e5e6-428e-9c58-02fce88615fe)